types/argparse/argparse-tests.ts

```diff
@@ -383,3 +384,14 @@ try {
     }
 }
 console.log('-----------');
+
+const booleanOptionalActionExample = new ArgumentParser();
+booleanOptionalActionExample.add_argument("--foo", { action: BooleanOptionalAction });
+booleanOptionalActionExample.print_help();
+console.log("-----------");
```

Comment:
Nit: The existing file uses single quotes for strings.  Would prefer to stay consistent within the file.

Reply:
hmm, the prettier options use double quotes. Should I follow the formatter, or the existing convention in this file?

Reply:
Oh, sorry, didn't realize there was a Prettier config.  I retract my comment then.